kanin/src/request.rs

```diff
@@ -25,6 +29,7 @@ impl Request {
         Self {
             state,
             channel,
+            req_id: ReqId::from(&delivery),
```

Comment:
I think the usage of the `From` trait here is slightly unnecessary (a simple `from_delivery` method would do) but it's not a big deal.

Reply:
I am actually not a super big fan of these impl's (as with the example with `thiserror`), so I'll be happy to change it :)